packages/integration-test-app/windows/integrationtest/TestCommand.h

```diff
@@ -0,0 +1,15 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+#pragma once
+
+#include <winrt/Windows.Data.Json.h>
```

Comment:
Unused include, or is this here to avoid having to declare it in consumers?

Reply:
Artifact from older file organization. Will remove